### PR TITLE
Code of conduct

### PR DESCRIPTION
**Pull request by nfmcclure**
Added code of conduct and contributing document. This merge should address the issues:
https://github.com/payscale/fables/issues/4
and
https://github.com/payscale/fables/issues/2

I also added a line to the `.gitignore` file to ignore the pycharm settings folder.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -10,3 +10,7 @@ __pycache__
 
 # Excel temporary files
 ~$*.xls*
+
+# Pycharm stuff
```
Comment (tojo13) [inline .gitignore]: Nice!

File: Code_of_Conduct.md [hunk 2]
```diff
@@ -28,10 +28,10 @@ Project maintainers are responsible for clarifying the standards of acceptable b
...
 
 ## Scope
-This Code of Conduct applies within all project spaces, and it also applies when an individual is representing the project or its community in public spaces. Examples of representing a project or community include using an official project e-mail address, posting via an official social media account, or acting as an appointed representative at an online or offline event. Representation of a project may be further defined and clarified by project maintainers.
+This Code of Conduct applies within all project spaces, and it also applies when an individual is representing the project or its community in public spaces. Examples of representing a project or community include using an official project e-mail address, posting via an official social media account, or acting as an appointed representative at an online or offline event. Representation of a project may be further defined and clarified by project maintainers (listed below).
```
Comment (tojo13) [inline Code_of_Conduct.md]: Minor nit. These longer lines might raise errors on someones markdown linter or be hard to read on some editors. Should we linebreak around ~100 lines?

Comment (nfmcclure) [reply Code_of_Conduct.md]: Sure, I don't mind.  I'll go through and add some line breaks.

**Comment by tojo13** [comment]
Thanks for doing this, Nick!

**Pull request merged**